style.css

```diff
@@ -24,14 +24,15 @@ header {
   font-style: normal;

```

Comment:
- [ ] In the body, the browser gives an automatically 8px margin, this creates an error in the design, you should add a CSS reset file to fix this. Check more [info here](https://stackoverflow.com/questions/12582624/what-is-a-user-agent-stylesheet)

![WhatsApp Image 2021-06-10 at 18 57 21](https://user-images.githubusercontent.com/60631456/121611570-28f66c80-ca1e-11eb-9c89-55f21805ca4d.jpeg)

![image](https://user-images.githubusercontent.com/60631456/121611597-3ad80f80-ca1e-11eb-8d16-5bae7c6d4322.png)

**Here is your [live demo](https://rawcdn.githack.com/Gunjuzone/My-Portfolio/4c4af3ee599686eb941e7a8791a0eaccd99b474d/index.html)**

Reply:
Hello, would something like this fix the error


* {  margin: 0;  padding: 0;  box-sizing: border-box; }
I dont really understand what the problem is

Reply:
> Hello, would something like this fix the error
> 
> * {  margin: 0;  padding: 0;  box-sizing: border-box; }
>   I dont really understand what the problem is

Open[ this link](https://rawcdn.githack.com/Gunjuzone/My-Portfolio/4c4af3ee599686eb941e7a8791a0eaccd99b474d/index.html) in a mobile phone, and you will see a margin in the body that doesn't need to be there. [Learn more](https://stackoverflow.com/questions/11578819/css-reset-what-exactly-does-it-do) about inconsistencies in browsers and how a CSS reset file can help you.